tools/ci/production-upgrade

```diff
@@ -52,3 +38,5 @@ UPGRADE_SCRIPT=/home/zulip/deployments/current/scripts/lib/upgrade-zulip
 
 # Execute the upgrade.
 sudo "$UPGRADE_SCRIPT" /tmp/zulip-server-test.tar.gz
+
+sudo supervisorctl restart
```

Comment:
This line doesn't do anything; from the logs:
```
+ sudo supervisorctl restart
Error: restart requires a process name
restart <name>		Restart a process
restart <gname>:*	Restart all processes in a group
restart <name> <name>	Restart multiple processes or groups
restart all		Restart all processes
Note: restart does not reread config files. For that, see reread and update.
```

Remind me what error are we're trying to resolve with it?

Reply:
The right call should have been `sudo supervisorctl restart all` which I actually forgot to update. This was a trial run to try to resolve the error rising from zulip workers failing to sustain after start in the production verify step.